container/karaf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -128,6 +128,7 @@ featuresBoot = ( \
   opennms-reporting, \
   opennms-search, \
   opennms-config-management, \
+  opennms-zenith-connect, \
```

Comment:
Do you want this features to be always enabled ?

Reply:
Removed.